server/services/totalCount.service.js

```diff
@@ -85,4 +96,83 @@ const getTotalCount = async (user) => {
   }
 };
 
-export default getTotalCount;
+export const getNeedAttention = async (user) => {
+  let response = {};
+
+  if (user.role === USER_ROLE.ADMIN) {
+    const pendingOffers = await Offer.countDocuments({
```

Comment:
Lets remove `pending` from the variable name (it is confusing for some of the variables that are not pending and would cause more harm than good in understanding the code in the future). Lets use the variable name directly, anyone consuming the endpoint will know the models that need attention. 

```suggestion
    const offers = await Offer.countDocuments({
```